static/app/actionCreators/pageFilters.tsx

```diff
@@ -469,6 +509,19 @@ export function updateDateTime(
   persistPageFilters('datetime', options);
 }
 
+/**
+ * Updates store and global repository selection URL param if `router` is supplied
+ *
+ * @param {Object} repository Object with repository key
+ * @param {Object} [router] Router object
+ * @param {Object} [options] Options object
```

Comment:
nit probably don't need to do jsdocs anymore like the others still have hanging around 
```suggestion
 * @param repository Object with repository key
 * @param router Router object
 * @param options Options object
```